src/features/ModelPlan/ModelToOperations/_components/ActionsMenu/index.tsx

```diff
@@ -0,0 +1,299 @@
+import React, { useContext, useEffect, useMemo, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { Button, Menu } from '@trussworks/react-uswds';
+import classNames from 'classnames';
+import i18next from 'i18next';
+
+import Modal from 'components/Modal';
+import PageHeading from 'components/PageHeading';
+import Sidepanel from 'components/Sidepanel';
+import { MTOModalContext } from 'contexts/MTOModalContext';
+import useCheckResponsiveScreen from 'hooks/useCheckMobile';
+import useMessage from 'hooks/useMessage';
+
+import EditMilestoneForm from '../EditMilestoneForm';
+import { MTORowType } from '../Table/columns';
+
+const ActionMenu = ({
+  rowType,
+  MoveUp,
+  MoveDown,
+  milestoneID,
+  primaryCategoryID,
+  subCategoryID
+}: {
```

Comment:
yass! thanks for moving this out to its own component file/folder. I was contemplating this when I was working with the action menu earlier.